docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -77,6 +77,12 @@ If you want to enable this feature, you'll need to enable the `onlyExternalOrgRo
 
 You can prevent the synchronization of organization roles from the authentication provider. For more information, refer to [skip org role sync]({{< relref "../setup-grafana/configure-grafana/#authgrafana_com-skip_org_role_sync/" >}}).
 
+## Removal of Creating API keys from the UI
```

Comment:
Suggestion.
```suggestion
## Removing the button to create API keys from the UI
```

Reply:
yea this makes it seem less aggressiveish 👍